template.yaml

```diff
@@ -182,9 +182,6 @@ objects:
         feedback_storage: "${STORAGE_MOUNT_PATH}/feedback"
         transcripts_enabled: ${LIGHTSPEED_TRANSCRIPTS_ENABLED}
         transcripts_storage: "${STORAGE_MOUNT_PATH}/transcripts"
-        data_collector:
```

Comment:
The default is false?

Reply:
Hmm not sure, it doesn't exist anymore, not sure what replaced it